kotlin-core/src/main/java/com/segment/analytics/kotlin/core/platform/Mediator.kt

```diff
@@ -1,10 +1,10 @@
-package com.segment.analytics.platform
+package com.segment.analytics.kotlin.core.platform
 
-import com.segment.analytics.*
+import com.segment.analytics.kotlin.core.*
 
 // Platform abstraction for managing plugins' execution (of a specific type)
 // All operations are thread safe via the `synchronized` function
-internal class Mediator(internal val plugins: MutableList<Plugin>) {
+class Mediator(internal val plugins: MutableList<Plugin>) {
```

Comment:
Love the mediator pattern ++!

Should we give it a more specific name?

Reply:
im actually going to move it back to being internal and expose the `applyClosure` function via `Analytics`